ci/build_python.sh

```diff
@@ -9,13 +9,27 @@ export CMAKE_GENERATOR=Ninja
 
 rapids-print-env
 
+package_name="cucim"
+package_dir="python/cucim/src"
+
+version=$(rapids-generate-version)
+# for CMake VERSION need to truncate any trailing 'a'
+version_cpp=${version%a*}
```

Comment:
suggestion (non-blocking): The alternative would be maintaining a single file and then stripping out the alpha in CMake. Now that it's done this way I don't know if it's worth changing, but it would let you carry around one less file.

Reply:
Agreed. I did it this way because I am not familiar with the proper CMake syntax to do it. Happy to update it if you have suggestions on how the CMake code would look.

Reply:
This would probably do it (after the existing `file(STRING...)` command):
```
string(REGEX REPLACE "a.*$" "" VERSION ${VERSION})                                                                                                                                                        
```

Reply:
Thanks, @vyasr, that seems to work! I pushed a commit to remove VERSION_CPP as well as simplify the CI scripts a bit (there was still writing of some unused VERSION file).

There should now be only a single top-level VERSION file (as well as  symlink to it from `python/cucim/src/cucim/VERSION`)